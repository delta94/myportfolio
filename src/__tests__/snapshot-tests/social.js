import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Social from '../../components/social';

it('Social renders correctly', () => {
  const wrapper = shallow(<Social />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
