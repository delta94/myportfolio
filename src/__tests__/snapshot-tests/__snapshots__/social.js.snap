// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Social renders correctly 1`] = `
<styled.div
  role="list"
>
  <styled.div
    role="listitem"
  >
    <styled.a
      href="http://www.linkedin.com/in/prabhu-murthy-09b36637"
      target="new"
      title="LinkedIn"
    >
      <img
        alt="linkedin"
        height="25px"
        src={[Function]}
        style={
          Object {
            "marginBottom": 0,
            "marginRight": "0.25rem",
          }
        }
        width="25px"
      />
    </styled.a>
  </styled.div>
  <styled.div
    role="listitem"
  >
    <styled.a
      href="http://github.com/prabhuignoto"
      target="new"
      title="Github"
    >
      <img
        alt="github"
        height="25px"
        src={[Function]}
        style={
          Object {
            "marginBottom": 0,
            "marginRight": "0.25rem",
          }
        }
        width="25px"
      />
    </styled.a>
  </styled.div>
  <styled.div
    role="listitem"
  >
    <styled.a
      href="http://stackoverflow.com/users/1329529/prabhu-murthy?tab=profile"
      target="new"
      title="StackOverflow"
    >
      <img
        alt="stackoverflow"
        height="25px"
        src={[Function]}
        style={
          Object {
            "marginBottom": 0,
            "marginRight": "0.25rem",
          }
        }
        width="25px"
      />
    </styled.a>
  </styled.div>
  <styled.div
    role="listitem"
  >
    <styled.a
      href="https://twitter.com/prabhumurthy2"
      target="new"
      title="Twitter"
    >
      <img
        alt="twitter"
        height="25px"
        src={[Function]}
        style={
          Object {
            "marginBottom": 0,
            "marginRight": "0.25rem",
          }
        }
        width="25px"
      />
    </styled.a>
  </styled.div>
  <styled.div
    role="listitem"
  >
    <styled.a
      href="mailto:prabhu.m.murthy@gmail.com"
      target="new"
      title="Get in Touch"
    >
      <img
        alt="stackoverflow"
        height="25px"
        src={[Function]}
        style={
          Object {
            "marginBottom": 0,
            "marginRight": "0.25rem",
          }
        }
        width="25px"
      />
    </styled.a>
  </styled.div>
</styled.div>
`;
